cherrypy/_cptree.py

```diff
@@ -217,6 +217,11 @@ def mount(self, root, script_name='', config=None):
                 'request). You cannot mount such Applications on this Tree; '
                 'you must pass them to a WSGI server interface directly.')
 
+        if script_name and not script_name.startswith('/'):
+            cherrypy.log.error(
+                msg='script_name {0} should start with /'.format(script_name)
+            )
+            script_name = '/' + script_name
```

Comment:
I'm not sure if it's a good idea for nested trees. Could you please check this too?